packages/runtime/src/client/client-impl.ts

```
@@ -199,6 +199,7 @@ export class ClientImpl<Schema extends SchemaDef> {
             );
         }
 
+        // eslint-disable-next-line @typescript-eslint/ban-types
         return (procOptions[name] as Function).apply(this, [this, ...args]);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)